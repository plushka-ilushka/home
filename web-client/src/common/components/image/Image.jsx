import React from 'react';

import PropTypes from 'prop-types';
import './image.scss';

export default function Image(props) {
  const {
    className,
    src,
    alt,
  } = props;

  return (
    <img
      className={className}
      src={src}
      alt={alt}
    />
  );
}

Image.propTypes = {
  className: PropTypes.string,
  src: PropTypes.string,
  alt: PropTypes.string,
};

Image.defaultProps = {
  className: undefined,
  src: '',
  alt: '',
};
