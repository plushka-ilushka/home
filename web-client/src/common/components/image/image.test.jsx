import React from 'react';
import { mount } from 'enzyme';
import Image from './Image';

describe('Image Component', () => {
  it('Image Snapshot', () => {
    expect(mount(<Image />)).toMatchSnapshot();
  });
});
