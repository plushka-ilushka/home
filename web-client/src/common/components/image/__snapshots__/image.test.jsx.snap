// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Component Image Snapshot 1`] = `
<Image
  alt=""
  src=""
>
  <img
    alt=""
    src=""
  />
</Image>
`;
